pom.xml

```diff
@@ -56,6 +56,12 @@
       <version>1.6.5</version>
     </dependency>
 
+    <!-- SESSION -->
+    <dependency>
+      <groupId>org.springframework.session</groupId>
+      <artifactId>spring-session-jdbc</artifactId>
+    </dependency>
+
```

Comment:
wie gehen wir generel mit problemen um, die durch mehrere instanzen des selben deployments verursacht werden? z.b. `@Scheduled`

Reply:
Das wird hier nicht berücksichtigt und muss man in einem gesonderten PR angehen. Hatte ich hier bewusst nicht berücksichtigt